packages/jest-haste-map/src/types.js

```diff
@@ -23,3 +24,11 @@ export type WorkerMetadata = {
   dependencies: ?Array<string>,
 };
 export type WorkerCallback = (error: ?Error, metaData: ?WorkerMetadata) => void;
+
+export type CrawlerOptions = {
```

Comment:
Could you elaborate? Just stumbled upon this looking over @matryoshcow’s shoulder. 
What is the use case for exact types?